apps/prairielearn/src/lib/format.test.ts

```diff
@@ -0,0 +1,76 @@
+import { assert } from 'chai';
+
+import { formatFloat, formatPoints, formatPointsOrList } from './format.js';
+
+describe('format library', () => {
```

Comment:
Not requesting a change, but it's also valid to have multiple top-level `describe` blocks (e.g. https://github.com/PrairieLearn/PrairieLearn/blob/54e09bba3606216478efeafe947c5c20b33a634b/apps/prairielearn/src/ee/models/enrollment.test.ts).

Reply:
Indeed I'm aware of that, but I typically find it easier to read test results if the top level describe refers to a specific test file.